.gitignore

```diff
@@ -13,3 +13,4 @@
 dist
 build
 venv*
+compose.yaml
```

Comment:
What's this for?

Reply:
I have a local docker compose file that springs up postgres, mysql, mssql, oracle and all other databases related to SQLAlchemy I use to test this library's generated bindings functional correctness. I want to make sure it doesn't acceidently pollutes my commits.

Reply:
Isn't that normally named `docker-compose.yml`?